pylintrc

```diff
@@ -0,0 +1,407 @@
+[MASTER]
+
+# Specify a configuration file.
```

Comment:
Sure. Pylint generates the config file and I used it as is. I'll remove the sections we don't use.